src/components/BlogFooter.astro

```diff
@@ -0,0 +1,60 @@
+---
+import { getPosts } from "@/lib/content";
+import { ArrowLeft, ArrowRight } from "lucide-react";
```

Comment:
I think we should keep lucide-react because those icons need to be used in both astro and react components. Passing the astro icons as react props is not the best DX and since we're not hydrating them, there should not be additional overhead. What do you think?

Reply:
Agree. I just want to get rid of React since it increases the bundle size, right?

Reply:
I think so. I was not sure how much you wanted the site to be migrated so good to know! After I addressed your reviews, I'll get rid of react